FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/ContextualCommandBarActivity.kt

```diff
@@ -34,86 +34,116 @@ class ContextualCommandBarActivity : DemoActivity() {
         // Used to test bitmap support
         var drawable = AppCompatResources.getDrawable(this, R.drawable.ic_fluent_add_24_regular)
         val bitmap: Bitmap = Bitmap.createBitmap(
-            drawable!!.getIntrinsicWidth(),
-            drawable!!.getIntrinsicHeight(),
+            drawable!!.intrinsicWidth,
```

Comment:
Double exclamation for intrinsic width but not for height?

Reply:
AndroidStudio Optimistaion. Basically !! is a force assert and if it passed in line 37, it's not required in line 38. If it fails in 37, the app crashes.